Improvements to Mini Surveys

AmrishBasdew opened the pull request:
Updated Mini Survey for the App Insights 2.16 release

AmrishBasdew committed:
Update mini-surveys.md

Updated Mini Surveys for the 2.16 release

AmrishBasdew committed:
Merge branch 'mendix:development' into development

AmrishBasdew committed:
Add files via upload

Added new Survey Details

AmrishBasdew committed:
Update mini-surveys.md

updated image for survey details

Luyao-Zhang-1 committed:
update images

Luyao-Zhang-1 commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -171,15 +145,18 @@ To edit a mini survey, perform the following steps:
    * If the mini survey has the status of **Draft**, you can edit everything in the mini survey. To edit the mini survey, click **Edit Draft** on the upper-right corner of the page, and then make changes.
-   * If the mini survey has the status of **Ready to Implement**, **Running**, or **Evaluate Responses**, you can only edit the **Settings** section (for **Runtime** and **Toaster Placement**) in the survey. Click **Edit Settings** and then make the changes.
+   * If the mini survey has the status of **Scheduled**, **Active**, or **Finished**, you can only edit the **Start and End Date** and the **Toaster Placement** in the survey. Click **Edit Settings** and then make the changes. Based on the dates set, the status of the survey will update:
+     * **Scheduled** if both the start and end dates are in the future.
```
@AmrishBasdew Do we need to repeat the description of the three statuses here? At the beginning of the bullet, there is a cross-reference to [status](#survey-status), which can direct users to the section "Statuses of Mini Surveys".

AmrishBasdew replied:
In line 152, the statuses are named because these are the only statuses where you can not edit the draft. We could rephrase it by saying 'If the mini survey doesn't have the **Draft** status..'

Luyao-Zhang-1 replied:
Got it. Let's keep it.

Luyao-Zhang-1 committed:
edit

Luyao-Zhang-1 committed:
Update mini-surveys.md

Luyao-Zhang-1 commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -142,25 +116,24 @@ To create a mini survey, follow these steps:
+        * **Multiple Choice** – Create a question where you can set up to 4 answers for the user to pick from.
+        * **NPS** – Let the user pick a score between 0 and 10 to rate your app.
+    * **Finalize** – Shows an summary of your survey.
+    * **Toaster Placement** – Set on which corner of the page the mini survey appears.
```
@AmrishBasdew Is **Toaster Placement** on the same page as **Finalize**, or is it on a separate page?

AmrishBasdew replied:
Ah, something went wrong with the formatting there! The page is called Finalize, and on this page you get the option to change the Toaster Placement. So Toaster Placement should have an extra indent.

AmrishBasdew committed:
Update mini-surveys.md

Fixed Luyao's findings

Luyao-Zhang-1 committed:
Update latest-releases.html

Luyao-Zhang-1 committed:
formatting

Luyao-Zhang-1 committed:
Merge branch 'development' of https://github.com/AmrishBasdew/docs into pr/8451

Luyao-Zhang-1 committed:
fix typo

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -7,15 +7,13 @@ description: "The app development team can utilize Mendix's user-friendly [Mini
 ## Introduction
 
-The Mini Surveys feature is part of the Mendix App Insights collection of tools designed to help you collect and aggregate feedback from your customers. 
+Mini Surveys is a part of Mendix App Insights, a set of tools designed to help you gather and consolidate feedback from your customers.
```
Change to "Mini Surveys is part of Mendix App Insights, a set of tools designed to help you aggregate feedback from your customers."

This more concise and cleaner english

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -35,8 +33,9 @@ The tabs of the **Survey Overview** page are described below.
 The **Active** tab shows all the mini surveys that are active for the app with the following details:
 
-* **Survey ID** – This is the unique ID of the mini survey. You can also find this ID on the [survey details](#survey-details) page of the mini survey. You need to use the ID as input when you configure the Mendix Mini Surveys module in Studio Pro to implement the mini survey.
+* **ID** – This is the unique ID of the mini survey. You can also find this ID on the [survey details](#survey-details) page of the mini survey. You need to use the ID as input when you configure the Mendix Mini Surveys module in Studio Pro to implement the mini survey.
```
Not used anymore in the configuration of module. This is deprecated in favour of Location variable.

Luyao-Zhang-1 replied:
@MichaelMendix Shall I just remove "Survey ID" here?

AmrishBasdew replied:
Good find Michael! As the ID still shows on the survey overview page, we should keep the entry. I've removed the second sentence as that part is outdated :)

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -35,8 +33,9 @@ The tabs of the **Survey Overview** page are described below.
-* **Survey ID** – This is the unique ID of the mini survey. You can also find this ID on the [survey details](#survey-details) page of the mini survey. You need to use the ID as input when you configure the Mendix Mini Surveys module in Studio Pro to implement the mini survey.
+* **ID** – This is the unique ID of the mini survey. You can also find this ID on the [survey details](#survey-details) page of the mini survey. You need to use the ID as input when you configure the Mendix Mini Surveys module in Studio Pro to implement the mini survey.
 * **Name** – This is the name of the mini survey.
+* **Location** - The Location is an identifier that refers to the location where the survey will show.
```
Needs more clarity where this survey is: Change to something like
"The Location is an identifier that refers to the location where the mini survey will pop-up in your web application."

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -171,15 +145,21 @@ To edit a mini survey, perform the following steps:
 
    * If the mini survey has the status of **Draft**, you can edit everything in the mini survey. To edit the mini survey, click **Edit Draft** on the upper-right corner of the page, and then make changes.
-   * If the mini survey has the status of **Ready to Implement**, **Running**, or **Evaluate Responses**, you can only edit the **Settings** section (for **Runtime** and **Toaster Placement**) in the survey. Click **Edit Settings** and then make the changes.
+   * If the mini survey does not have the status of **draft**, you can only edit the **Start and End Date** and the **Toaster Placement** in the survey. Click **Edit Settings** and then make the changes. Based on the dates set, the status of the survey will update as follows:
```
Confusing english. Better to explain clearly that If the survey has status of "active" or "Finished" you can only edit the start and end date.

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -226,7 +203,6 @@ To check the responses that have been collected, follow these steps:
 A mini survey has the following opt-out rules: 
```
I  was on holiday when opt-out rules were changed. Are these rules still valid?

AmrishBasdew replied:
I've updated these rules!

MichaelMendix commented on content/en/docs/developerportal/app-insights/mini-surveys.md:
```diff
@@ -226,7 +203,6 @@ To check the responses that have been collected, follow these steps:
 A mini survey has the following opt-out rules: 
```
Should mention the demo survey with "Default_location" only appears once.

Luyao-Zhang-1 replied:
What does this mean exactly?

AmrishBasdew replied:
I've added a mention to the demo survey in the Create new Survey flow

Luyao-Zhang-1 committed:
update based on feedback

AmrishBasdew committed:
Update mini-surveys.md

Updated according to Michael's comments

Luyao-Zhang-1 committed:
Update latest-releases.html

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/8451

Luyao-Zhang-1 committed:
Update _index.md

Luyao-Zhang-1 committed:
Language edit

AmrishBasdew committed:
Update mini-surveys.md

Added info to explain that changes made to an active survey can take up to 10 minutes to show in the app

AmrishBasdew committed:
Update mini-surveys.md

Updated the Opt-out rules

Luyao-Zhang-1 committed:
fix indentation

AmrishBasdew committed:
Update mendix-mini-surveys.md

Added warning to the top of the page

AmrishBasdew committed:
Update mendix-mini-surveys.md

Updated the release version

AmrishBasdew committed:
Update mendix-mini-surveys.md

Updated version

Luyao-Zhang-1 committed:
edit

Luyao-Zhang-1 committed:
Merge branch 'development' of https://github.com/AmrishBasdew/docs into pr/8451

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/8451

Luyao-Zhang-1 committed:
fix the broken link

Luyao-Zhang-1 committed:
correct typo in image name
